app/views/full-page-examples/search/styles.scss

```diff
@@ -0,0 +1,88 @@
+@import "../../../../src/govuk/base";
+
+.full-page-example__search {
+  $govuk-new-link-styles: true;
+  @import "../../../../src/govuk/core/all";
+
+  .app-document-list {
+    margin-top: 0;
+    margin-bottom: 0;
+    padding: 0;
+    list-style: none;
+  }
+
+  .app-document-list > li {
+    padding-top: govuk-spacing(4);
+    padding-bottom: govuk-spacing(2);
+    border-bottom: 1px solid $govuk-border-colour;
+  }
+
+  .app-metadata-list {
+    margin: 0;
+    padding: 0;
+    list-style: none;
+  }
+
+  .app-metadata-list > li {
+    display: inline-block;
+  }
+
+  .app-filter li {
+    padding-bottom: govuk-spacing(4);
+    border-bottom: 1px solid $govuk-border-colour;
+  }
+
+  .govuk-fieldset__legend {
+    margin-top: govuk-spacing(4);
+    margin-bottom: govuk-spacing(4);
+    font-weight: normal;
+  }
+
+  .app-organisation-checkboxes {
+    padding-bottom: govuk-spacing(4);
+    border-bottom: 1px solid govuk-colour("mid-grey");
+  }
+
+  .app-search-result-list__children {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    column-gap: 15px;
+    list-style: none;
+  }
```

Comment:
The behaviour on GOV.UK is slightly different on mobile and tablet

<img width="476" alt="Screenshot 2021-05-26 at 09 25 12" src="https://user-images.githubusercontent.com/121939/119627837-75c62a80-be04-11eb-8345-58512182369e.png">
<img width="547" alt="Screenshot 2021-05-26 at 09 25 22" src="https://user-images.githubusercontent.com/121939/119627848-76f75780-be04-11eb-8f64-3fdaea50913a.png">

Don't think we need to replicate it exactly, but we probably do want to revert to a single column for mobile, and maybe tablet?